eng/common/scripts/Detect-Api-Changes.ps1

```diff
@@ -61,8 +61,7 @@ function Should-Process-Package($pkgPath, $packageName)
     # Get package info from json file created before updating version to daily dev
     $pkgInfo = Get-Content $pkgPropPath | ConvertFrom-Json
     $packagePath = $pkgInfo.DirectoryPath
-    $modifiedFiles = git diff --name-only --relative $TargetBranch HEAD
-    $modifiedFiles = $modifiedFiles.Where({$_.startswith($packagePath)})
+    $modifiedFiles  = Get-PullRequest-Diff-Changes -DiffPath "$packagePath/*" -DiffFilterType ''
```

Comment:
I assume `$packagepath/*` is to handle the `--relative`, correct?

Reply:
No,
`relative` is used to return the git diff path relative to current folder.
I believe the script executes under the working directory, so it is redundant to have to me. 
@praveenkuttappan Could you please verify the needs of the `--relative`?

$packagePath is the path to the package.
e.g. https://github.com/Azure/azure-docs-sdk-java/blob/main/metadata/preview/azure-ai-formrecognizer.json#L5
The API changes only care the files under package repo, so it is a filter to diff files.